### PR TITLE
[jsweb] self-host the kitty gif #336

### PR DESCRIPTION
**Pull request by evilzero**
# Summary

Fixes issue #336 

Here's a description of changes:

* uploaded cat-walk.gif
* modified 1 link on animation.html
* modified 1 link on catwalk.html

cc @gdisf/admins

### CODE REVIEW

**Comment by verythorough** [comment]
Deploy preview for *teaching-materials* ready!

Built with commit afa3dbc05027c5bcf948a3d5a066a4042a6989f7

https://deploy-preview-620--teaching-materials.netlify.com

File: jsweb/exercises/animation.html [hunk 1]
```diff
@@ -28,7 +28,7 @@ <h3 class="page-header">The Cat Walk</h3>
  &lt;/head&gt;
  &lt;body&gt;
```
Comment (trucnguyen) [inline jsweb/exercises/animation.html]: Nice, let's add a credit / attribution to the original source.

File: jsweb/exercises/catwalk.html [hunk 2]
```diff
@@ -30,7 +30,7 @@ <h2>Exercise: The Cat Walk</h2>
   &lt;button id=&quot;stop-button&quot;&gt;Stop&lt;/button&gt;
   &lt;div id=&quot;info&quot;&gt;&lt;/div&gt;
   
-  &lt;img style=&quot;position:absolute; left: 0px; top: 80px;&quot; src=&quot;http://www.anniemation.com/clip_art/images/cat-walk.gif&quot;&gt;
```
Comment (trucnguyen) [inline jsweb/exercises/catwalk.html]: Same here - credit anniemation.com

File: accessibility/marketing.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+## One Paragraph Announcement
```
Comment (trucnguyen) [inline accessibility/marketing.md]: Add the accessibility marketing.md should not be part of this PR
